test/controllers/feeds_controller_test.rb

```diff
@@ -5,21 +5,35 @@ class FeedsControllerTest < ActionDispatch::IntegrationTest
   let(:feed) { create(:feed, podcast: podcast, private: false) }
   let(:private_feed) { create(:private_feed, podcast: podcast) }
   let(:update_params) { {url: "/a_public_url", display_episodes_count: 5} }
+  let(:create_params) { {podcast: podcast, slug: "new_feed", title: "new title", private: false} }
 
   setup_current_user { build(:user, account_id: 123) }
 
-  # test "should get new" do
-  #   get new_feed_url
-  #   assert_response :success
-  # end
+  test "should get new" do
```

Comment:
I'd probably also add a validation test to this - "authorizes new feed" version of this.